presto-main/etc/catalog/gsheets.properties

```diff
@@ -0,0 +1,20 @@
+############ How to generate google-sheets service account credentials ################
```

Comment:
Instead of putting this here, let's create README.md file on top of `presto-google-sheets` which informations like that. I don't think this is easily findable. The next step would be to have proper documentation.